imagetest/cmd/manager/main.go

```diff
@@ -35,6 +35,7 @@ var (
 	parallelCount = flag.Int("parallel_count", 5, "TestParallelCount")
 	filter        = flag.String("filter", "", "only run tests matching filter")
 	exclude       = flag.String("exclude", "", "skip tests matching filter")
+	machineType   = flag.String("machine_type", "", "machine type")
```

Comment:
I guess machine type for VMs running the test? Extra description would be useful.